components/styled/table/src/atoms/table.tsx

```diff
@@ -8,6 +8,18 @@ const Container = styled.div`
   background-color: white;
   border-radius: 6px;
   display: inline-block;
+  max-width: calc(100%);
+  max-height: calc(100%);
+  overflow: scroll;
+  &::-webkit-scrollbar {
```

Comment:
I can be convinced since it appears to already have support in all major browsers & will just fall back if not available, but these are not technically standard features in the css spec and are open to change so introducing them here could require further work should the feature definition change in the future

https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

Reply:
![image](https://user-images.githubusercontent.com/115567223/199450861-9d793458-d945-4f0a-9106-22420cedc862.png)

Tried to match the design on miro board. shall I revert it back to standard scrollbar?

Reply:
Styling isn't my fort, but as far as I can tell caniuse.com says it won't work on Firefox or IE 11 which I think the hospitals still use. 
Did you get chance to run it through Browserstack, @saqib-airelogic ? If not I'd suggest doing that next to give us some confidence it'll work on hospital devices. 
One of us should probably pick up that accessibility ticket next I guess, happy to take a look unless one of you fancies it :)

Reply:
Be interested to see it on Browserstack, my guess is it will fall back to the standard scrollbars (CSS spec says ignore any statements you can't process).

Happy to pick up some work on this next week if there's anything left by then!

Reply:
Airelogic have a shared account, you'll need to use the creds for that. They're saved in Bitwarden :)

Reply:
Just checked IE and it falls back to the normal scrollbars so I think this is OK to do so long as we're aware at some point the specification may change and it'll need revisiting